coconut/command/command.py

```diff
@@ -277,7 +284,7 @@ def cmd(self, args, interact=True):
         try:
             if args.recursion_limit[0] is not None:
                 sys.setrecursionlimit(args.recursion_limit[0])
-            self.setup(args.target[0], args.strict, args.minify, args.line_numbers, args.keep_lines, args.quiet, args.color[0])
+            self.setup(args.target[0], args.strict, args.minify, args.line_numbers, args.keep_lines, args.quiet, args.color[0], args.backwards)
```

Comment:
Can you match how `args.color` works and take the argument out of the array before passing it to `setup`?